packages/ez-react/src/docs/4_3_commonProps.stories.mdx

```diff
@@ -94,50 +94,81 @@ We use the ones supplied by [d3-ease](https://github.com/d3/d3-ease) library for
 So, please refer to the following guide to get a grasp of how each easing function would animate:
 [Transition Easing Comparison in v4](https://bl.ocks.org/d3noob/1ea51d03775b9650e8dfd03474e202fe)
 
-Available easing functions are : <animation_name_used_in_eachychart> : <type_of_the_animation>
+Available easing functions are :
 
-easeLinear: 'Linear’ | easeQuad: 'Quad' | easeQuadIn: 'Quand In' | easeQuadOut: 'Quad Out' | easeQuadInOut: 'Quad In/Out' |
-easeCubic: 'Cubic' | easeCubicIn: 'Cubic In' | easeCubicOut: 'Cubic Out' | easeCubicInOut: 'Cubic In/Out' |
-easePoly: 'Polynomial' | easePolyIn: 'Polynomial In' | easePolyOut: 'Polynomial Out' | easePolyInOut: 'Polynomial In/Out' |
-easeSin: 'Sinusoidal' | easeSinIn: 'Sinusoidal In' | easeSinOut: 'Sinusoidal Out' | easeSinInOut: 'Sinusoidal In/Out' |
-easeExp: 'Exponential' | easeExpIn: 'Exponential In' | easeExpOut: 'Exponential Out' | easeExpInOut: 'Exponential In/Out' |
-easeCircle: 'Circle' | easeCircleIn: 'Circle In' | easeCircleOut: 'Circle Out' | easeCircleInOut: 'Circle In/Out' |
-easeBounce: 'Bounce' | easeBounceIn: 'Bounce In' | easeBounceOut: 'Bounce Out' | easeBounceInOut: 'Bounce In/Out' |
-easeBack: 'Back' | easeBackIn: 'Back In' | easeBackOut: 'Back Out' | easeBackInOut: 'Back In/Out' | easeElastic: 'Elastic' |
-easeElasticIn: 'Elastic In' | easeElasticOut: 'Elastic Out' | easeElasticInOut: 'Elastic In/Out'.
+`<animation_name_used_in_eazychart>` : <type_of_the_animation>
+
+`easeLinear`: 'Linear’ | `easeQuad`: 'Quad' | `easeQuadIn`: 'Quand In' | `easeQuadOut`: 'Quad Out' | `easeQuadInOut`: 'Quad In/Out' |
+`easeCubic`: 'Cubic' | `easeCubicIn`: 'Cubic In' | `easeCubicOut`: 'Cubic Out' | `easeCubicInOut`: 'Cubic In/Out' |
+`easePoly`: 'Polynomial' | `easePolyIn`: 'Polynomial In' | `easePolyOut`: 'Polynomial Out' | `easePolyInOut`: 'Polynomial In/Out' |
+`easeSin`: 'Sinusoidal' | `easeSinIn`: 'Sinusoidal In' | `easeSinOut`: 'Sinusoidal Out' | `easeSinInOut`: 'Sinusoidal In/Out' |
+`easeExp`: 'Exponential' | `easeExpIn`: 'Exponential In' | `easeExpOut`: 'Exponential Out' | `easeExpInOut`: 'Exponential In/Out' |
+`easeCircle`: 'Circle' | `easeCircleIn`: 'Circle In' | `easeCircleOut`: 'Circle Out' | `easeCircleInOut`: 'Circle In/Out' |
+`easeBounce`: 'Bounce' | `easeBounceIn`: 'Bounce In' | `easeBounceOut`: 'Bounce Out' | `easeBounceInOut`: 'Bounce In/Out' |
+`easeBack`: 'Back' | `easeBackIn`: 'Back In' | `easeBackOut`: 'Back Out' | `easeBackInOut`: 'Back In/Out' | `easeElastic`: 'Elastic' |
+`easeElasticIn`: 'Elastic In' | `easeElasticOut`: 'Elastic Out' | `easeElasticInOut`: 'Elastic In/Out'.
```

Comment:
The animation list does not show properly. Please use a markdown table generator : https://www.tablesgenerator.com/markdown_tables

Reply:
Done :)